docs/tabs.md

```diff
@@ -228,3 +228,30 @@ class-label
 
 class-content
 : Additional CSS classes for the content element.
+
+## Synchronised tab options
+
+### URL parameter
+
+By default, the URL does not indicate the tab that you selected.
+To include the tab as a URL parameter, add a setting like the following example in your `conf.py` file:
+
+```python
+sphinx_design_sync_tabs_url_param = "tab"
+```
+
+With a setting like the preceding example, after you browse to a URL like `https://www.example.com/project-A/index.html` and select a tab that you configured with the `sync` option of `python`, the URL changes to `https://www.example.com/project-A/index.html?tab=python`.
+
+### Local storage key
+
+Synchronised tabs and tabbed code examples store the value of the `sync` option in browser local storage so that a return visit to a page opens the most-recently viewed tab.
+The storage applies to a single domain, such as `https://www.example.com`.
```

Comment:
```suggestion
This applies to all pages in a single domain, such as `https://www.example.com`.
```

Reply:
Unless you feel strongly, my preference is to restate the noun with "The storage" and I'm glad to add the "all pages in..."  The "all pages" adds a lot of clarify in very few words.

Reply:
yep that works for me!